src/zolltools/db/pqtools.py

```diff
@@ -106,6 +107,35 @@ def _get_parquet_file(self, pq_name: str, tmp=True) -> tuple:
 
         return (pq.ParquetFile(file_path), file_path)
 
+    def _calc_row_size(self, pq_file: pq.ParquetFile) -> int:
+        """
+        Returns the size (in bytes) of a row of a parquet file when converted to
```

Comment:
Double check whether it is in bytes or now.

Reply:
[Confirmed](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.memory_usage.html)